src/tasks/installer.tasks/GenerateRunScript.cs

```diff
@@ -62,25 +62,34 @@ private void WriteRunScript(string templateContent, string extension)
         {
             bool isUnix = extension == ".sh";
             string lineFeed = isUnix ? "\n" : "\r\n";
+            string[] newlineSeparator = new string[] { Environment.NewLine };
```

Comment:
It turns out that I don't use the changes in this file anymore (I did in an intermediate stage). But it seems like a useful change to take anyway, as currently the generated commands don't handle multi-line commands.